packages/web/src/common/store/tipping/sagas.ts

```diff
@@ -376,12 +376,23 @@ function* sendTipAsync() {
         source
       })
     )
-    yield put(refreshTipGatedTracks({ userId: recipient.user_id, trackId }))
-    yield fork(function* () {
+
+    yield* put(refreshTipGatedTracks({ userId: recipient.user_id, trackId }))
+    yield* fork(function* () {
       yield* call(confirmTipIndexed, { sender, recipient })
       yield* put(
         fetchPermissions({ userIds: [sender.user_id, recipient.user_id] })
       )
+      if (source === 'inboxUnavailableModal') {
+        console.debug('Creating chat silently...')
+        // Create the chat but don't navigate
+        yield* put(
+          chatActions.createChat({
+            userIds: [recipient.user_id],
+            skipNavigation: true
```

Comment:
Just curious - why do we need to skip navigation? If we're already in the right chat (with loading spinner) and we navigate to it _again_, does it look bad?

Reply:
The user's still looking at the tip sent screen. We don't want to navigate unless they close it